cif2cell/cif2cell.xml

```diff
@@ -16,12 +16,99 @@
         <requirement type="package" version="2.0.0a3">cif2cell</requirement>
     </requirements>
     <command detect_errors="exit_code"><![CDATA[
-        file_name=\$(sed 's/ //g' <<< '$file.name') &&
-        ln -s '$file' \$file_name &&
-        cif2cell -f \$file_name -p castep -o out.cell
+        #if $use_no_reduce_conditional.use_no_reduce == "true":
+            #if $vca_conditional.vca == "false":
+                #if $cartesian_conditional.cartesian == "false":
+                    file_name=\$(sed 's/ //g' <<< '$file.name') &&
+                    ln -s '$file' \$file_name &&
+                    cif2cell -f \$file_name -p castep -o out.cell --no-reduce
+                #end if
+            #end if
+        #end if
+        #if $use_no_reduce_conditional.use_no_reduce == "false":
+            #if $vca_conditional.vca == "false":
+                #if $cartesian_conditional.cartesian == "false":
+                    file_name=\$(sed 's/ //g' <<< '$file.name') &&
+                    ln -s '$file' \$file_name &&
+                    cif2cell -f \$file_name -p castep -o out.cell 
+                #end if
+            #end if
+        #end if
+        #if $use_no_reduce_conditional.use_no_reduce == "true":
+            #if $vca_conditional.vca == "true":
+                #if $cartesian_conditional.cartesian == "true":
+                    file_name=\$(sed 's/ //g' <<< '$file.name') &&
+                    ln -s '$file' \$file_name &&
+                    cif2cell -f \$file_name -p castep -o out.cell --no-reduce --vca --cartesian
```

Comment:
If we use booleans instead, we can simplify this to:
`cif2cell -f \$file_name -p castep -o out.cell $no_reduce $vca $cartesian`